scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -709,26 +705,31 @@ class Router(formatOps: FormatOps) {
           else if (right.is[T.LeftBrace]) NoSplit
           else Newline
 
-        val charactersInside = (close.start - open.end) - 2
-
         val defnSite = isDefnSite(leftOwner)
         val expirationToken: Token =
           if (defnSite && !isBracket) defnSiteLastToken(open, leftOwner)
           else rhsOptimalToken(leftTok2tok(close))
+        val newlineExpirationTokenOption =
+          Some(expirationToken)
+            .filterNot(_.is[T.Comment] && style.activeForEdition_2019_11)
```

Comment:
I had an edition guard here, but perhaps it makes sense to move it into the `rhsOptimalToken` method, plus I had forgotten that we released already and the edition might need to be incremented.

N.B. I know there was a discussion earlier about the difference between editions and releases but I am not yet too clear on that... if a release forces one to change the edition, they seem to be just two different ways of accomplishing the same.